src/Raven.Server/Documents/ETL/EtlProcess.cs

```diff
@@ -554,6 +560,25 @@ public override EtlPerformanceStats[] GetPerformanceStats()
                 .ToArray();
         }
 
+        private void LogSuccessfulBatchInfo(EtlStatsScope stats)
+        {
+            var message = new StringBuilder();
+
+            message.Append(
+                $"{Tag} process '{Name}' extracted {stats.NumberOfExtractedItems} docs, transformed and loaded {stats.NumberOfTransformedItems} docs in {stats.Duration}. ");
+
+            message.Append($"{nameof(stats.LastTransformedEtag)}: {stats.LastTransformedEtag}. ");
+            message.Append($"{nameof(stats.LastLoadedEtag)}: {stats.LastLoadedEtag}. ");
+
+            if (stats.LastFilteredOutEtag > 0)
+                message.Append($"{nameof(stats.LastFilteredOutEtag)}: {stats.LastFilteredOutEtag}. ");
+
+            if (stats.BatchCompleteReason != null)
+                message.Append($"Batch completion reason: {stats.BatchCompleteReason}");
+
+            Logger.Info(message.ToString());
```

Comment:
You checked operations, logged info (need to do that on info)

Reply:
Thanks. Fixed in https://github.com/ravendb/ravendb/pull/3881